.circleci/config.yml

```diff
@@ -98,6 +98,27 @@ jobs:
                   name: Tag and push image
                   command: docker tag ci-ubuntu-jammy $TARGETNAME/ci-ubuntu-jammy && docker push $TARGETNAME/ci-ubuntu-jammy
 
+  ubuntu-latest:
+      docker:
+        - image: openquantumsafe/ci-ubuntu-focal-x86_64:latest
```

Comment:
Why not also use ubuntu:latest as the build image? Otherwise we won't be able to get rid of focal...

Reply:
Line 82 above uses openquantumsafe/ci-ubuntu-focal-x86_64:latest as the build image for jammy. I was confused so I decided to copy it ;)

Now it's fixed